### PR TITLE
Confirm if pushing to master

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

asks you to confirm if you're pushing to master

## What is the value of this and can you measure success?

stop inadvertent pushes to master

## Screenshots

screen shot 2017-10-11 at 11 50 29

/cc @JonNorman @katebee @rich-nguyen

### CODE REVIEW

File: git-hooks/pre-push [hunk 1]
```diff
@@ -5,6 +5,36 @@ const path = require('path');
 const execa = require('execa');
 const chalk = require('chalk');
 
+const confirmIfMasterMessage = `${chalk.red(
+    `${chalk.inverse(
+        'You are about to push master!'
+    )}\nIs that what you intended?`
+)} [y/N]`;
+const confirmIfMaster = execa
+    .shell(
```
Comment (sndrs) [inline git-hooks/pre-push]: bash in js! yay!

Comment (SiAdcock) [reply git-hooks/pre-push]: 😲

**Review by SiAdcock — APPROVED** [review]
Amazing 👍 🎖 👏

File: git-hooks/pre-push [hunk 2]
```diff
@@ -5,6 +5,36 @@ const path = require('path');
...
+        `
+        protected_branch='master'
+        current_branch=$(git symbolic-ref HEAD | sed -e 's,.*/\\(.*\\),\\1,')
```
Comment (mchv) [inline git-hooks/pre-push]: `git rev-parse --abbrev-ref HEAD` should be faster and more reabable

Comment (sndrs) [reply git-hooks/pre-push]: oh nice one, cheers

**Comment by zeftilldeath** [comment]
End of an era

**Pull request merged**